src/Symfony/Component/Form/FormEvents.php

```diff
@@ -11,6 +11,12 @@
 namespace Symfony\Component\Form;
 
 /**
+ * To learn more about how form events work check the documentation
+ * entry at {@link http://symfony.com/doc/current/components/form/form_events.html}
```

Comment:
No I would add it to the `/current/` pages only if they are shown via an `/any/` url. Although maybe a permanent one would make sense too, I don't know if people typically get confused or not.